packages/camera/camera_avfoundation/example/ios/RunnerTests/Mocks/MockCaptureDevice.m

```diff
@@ -101,11 +100,37 @@ - (BOOL)isExposureModeSupported:(AVCaptureExposureMode)mode {
   return self.exposureModeSupported;
 }
 
-- (AVCaptureInput *)createInput:(NSError *_Nullable *_Nullable)error {
-  if (self.createInputStub) {
-    return self.createInputStub(error);
+@synthesize device;
```

Comment:
what is this for?

Reply:
Conforming to a protocol that declares properties doesn't trigger auto-synthesis (unlike explicitly declaring a property in a class), so this is necessary to get what we would generally think of as default implementations of the properties that come from the protocols these mocks implement.